addons/event_sale/views/event_views.xml

```diff
@@ -26,4 +26,71 @@
         </field>
     </record>
 
+    <record id="view_event_sale_report_graph" model="ir.ui.view">
```

Comment:
Guideslines: id should be ``event_sale_report_view_graph`` (valid for other views)